assets/readme/openjdk-white.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Hi @zrfisaac 

You need to use the same content of `openjdk.svg` but adding `fill="white"` attribute or [the README test will not pass](https://github.com/simple-icons/simple-icons/runs/6340479990?check_suite_focus=true). See the other icons in  [assets/readme](https://github.com/simple-icons/simple-icons/tree/b2cf3885ce82351a74b3562c7a19b9e7a45ce67b/assets/readme) directory as reference. Thanks!

Reply:
Thanks I fogot to do that, I will fix today!

Reply:
But please, separate the logo change in the third party plugin from this PR so we can update it without a major version release, see https://github.com/simple-icons/simple-icons/pull/7378#issuecomment-1120407819